bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: clowder
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.12.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.6.1+git
```

Comment:
This doesn't seem intentional and is maybe some local dev environment artifact, yeah?

Reply:
Yeah accidental commit, fixed